askama_actix/src/lib.rs

```diff
@@ -12,6 +27,24 @@ pub trait TemplateToResponse {
     fn to_response(&self) -> HttpResponse<BoxBody>;
 }
 
+struct WrappedError(Error);
```

Comment:
What problem is this solving?

Reply:
```rs
HttpResponse::from_error(Box::new(err) as Box<dyn std::error::Error + 'static>)
```
This line gives a "trivial_casts" warning, most likely a false positive, because `as Box<…>` cannot be omitted or  replaced by `as _`.

But actually WrappedError is a better implementation, because it removes the need for `dyn` and `Box`.

Reply:
Well, let's discuss that change in a separate PR.

Reply:
Let's rename it to `ActixError` and add a comment that this is just a newtype wrapper that implements `actix_web::ResponseError`?